dars/clearlinux/mkl/Dockerfile

```diff
@@ -34,10 +37,10 @@ RUN URL='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15816' &
 # file exists on different layers. To minimize docker
 # image size, remove the overlapped files before copy.
 RUN mkdir /os_core_install
-COPY --from=clearlinux/os-core:latest / /os_core_install/
+COPY --from=base-image / /os_core_install/
```

Comment:
Are you copying all root (/) from "base-image"? I yes, then you might not need all other COPYs in later layers. You can also avoid copying all root and copy tiny pieces instead.

Reply:
Make sense